packages/grid/x-data-grid/src/locales/itIT.ts

```diff
@@ -122,7 +123,7 @@ const itITGrid: Partial<GridLocaleText> = {
   // Column pinning text
   pinToLeft: 'Blocca a sinistra',
   pinToRight: 'Blocca a destra',
-  unpin: 'Sblocca',
+  // unpin: 'Unpin',
```

Comment:
Was the previous translation wrong?

Reply:
Oops, overridden accidentally, thanks for pointing out.